src/templates/license-manager.defaults.template

```diff
@@ -1,7 +1,9 @@
 LM2_AGENT_BACKEND_BASE_URL={{ license_manager_backend_base_url }}
 LM2_AGENT_LOG_BASE_DIR={{ log_base_dir }}
 LM2_AGENT_LOG_LEVEL={{ log_level }}
+{% if sentry_dsn %}
```

Comment:
This implies that the LM2_AGENT_SENTRY_DSN might not be set .... while in the other PR we ensure that it must be set and an exception will be thrown otherwise. Do we want to make this value mandatory here to match the implied configuration of the license-manger agent itself?

Reply:
Or do we want to modify the license-manager agent to support not configuring it?

Reply:
I see. What if we make this config defaults to an empty string? This way we can use the default value when running the agent locally and it won't crash the code.